GAE/src/org/akvo/flow/rest/dto/ReportDto.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright (C) 2018 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.flow.rest.dto;
+
+import java.util.Date;
+
+import com.gallatinsystems.framework.gwt.dto.client.BaseDto;
+
+public class ReportDto extends BaseDto {
+
+    private static final long serialVersionUID = 3197857074377585732L;
+
+    private String reportType;
+    private Long formId;
+    private String state;
+    private String filename;
+    private String message;
```

Comment:
Is this the same `message` that goes to the `Messages` tab?

Reply:
It's more details for the state, especially if the state is FINISHED_ERROR.